assets/batch_score/components/driver/src/batch_score_oss/utils/v2_output_formatter.py

```diff
@@ -74,7 +74,8 @@ def format_output(self, results: "list[ScoringResult]", batch_size_per_request:
         return list(map(self._stringify_output, output_list))
 
     def __get_request_id(self, scoring_request: ScoringResult):
-        return scoring_request.response_headers.get("x-request-id", "")
+        if scoring_request.response_headers is not None:
```

Comment:
I am not a python expert. What will this return in case of `scoring_request.response_headers is null`?

Reply:
thanks for noticing this one. Updated